cookiecutter-xblock/{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -27,64 +27,47 @@ max-line-length = 120
 ; D412 = No blank lines allowed between a section header and its content (numpy style)
 ; D413 = Missing blank line after last section (numpy style)
 ; D414 = Section has no content (numpy style)
-ignore = D101,D200,D203,D212,D215,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
+ignore = D101,D200,D203,D212,D215,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414,D401,D205 
 match-dir = (?!migrations)
 
 [pytest]
-DJANGO_SETTINGS_MODULE = translation_settings
-addopts = --cov {{ cookiecutter.package_name }} --cov-report term-missing --cov-report xml
+DJANGO_SETTINGS_MODULE = myxblock.settings.test
+addopts = --cov myxblock --cov-report term-missing --cov-report xml
```

Comment:
Why are we hardcoding this?

Reply:
It automatically hardcodes the name of xblock and replaces `{{ cookiecutter.package_name }}` which in my case is `myxblock` , once you use this xblock template to create a new xblock. (This is one of the initial steps)